modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -241,18 +292,18 @@ const OrderableTable = ({
 	className,
 	creationMenuItems,
 	creationMenuLabel = Liferay.Language.get('add'),
-	disableSave,
 	fields,
 	items: initialItems,
 	noItemsButtonLabel,
 	noItemsDescription,
 	noItemsTitle,
-	onCancelButtonClick,
 	onOrderChange,
-	onSaveButtonClick,
 	title,
 }: IOrderableTableProps) => {
 	const [items, setItems] = useState(initialItems);
+	const [order, setOrder] = useState(
+		initialItems.map((item) => item.id).join(',')
+	);
```

Comment:
This is used to track if we reordered back to initial state. If we did, there is no need for update request.